src/js/models/maps/viewfinder/ExpansionPanelsModel.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+define([], () => {
+  /**
+  * @class ExpansionPanelsModel
+  * @classdes ExpansionPanelsModel maintains state for multiple
+  * ExpansionPanelView instances so that only one is open at a time.
+  * @classcategory Models/Maps
+  */
+  const ExpansionPanelsModel = Backbone.Model.extend({
+    /**
+     * @name ExpansionPanelsModel#defaults
+     * @type {Object}
+     * @property {ExpansionPanelView[]} panels The expansion panel views that
+     * are meant to have only a single panel open at a time.
+     * @property {boolean} isMulti Whether multiple panels can be open at the
+     * same time when displayed in a group of panels. 
+     */
+    defaults() {
+      return {
+        panels: [],
+        isMulti: false,
+      }
+    },
+
+    /**
+     * @param {boolean} isMulti The display mode of the panels.
+     */
+    initialize({ isMulti } = { isMulti: false }) {
+      this.set('isMulti', isMulti);
```

Comment:
attributes passed to the model will [automatically be set](https://backbonejs.org/#Model-constructor). Is this needed? Is it intentional to prevent any additional attributes from being set (e.g. panels)?

Reply:
Nope, if its not necessary, I'll remove.